.github/workflows/npm-ecdsa.yml

```diff
@@ -0,0 +1,55 @@
+name: NPM ECDSA
+
+on:
+  push:
+    branches:
+      - main
+      - npm-ecdsa # TODO: remove before merging to main
+    paths:
+      - "solidity/ecdsa/contracts**"
```

Comment:
Let's also add `"solidity/ecdsa/deploy/**"` directory as we may use the deployment scripts in other projects.